api/bases/neutron.openstack.org_neutronapis.yaml

```diff
@@ -2092,6 +2092,14 @@ spec:
                       to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/'
                     type: object
                 type: object
+              routeAnnotations:
+                description: RouteAnnotations to be added to the created route
+                properties:
+                  timeout:
+                    default: "60"
```

Comment:
Maybe we don't want to have a default at all? 
As a route will already have a 30s timeout from HAProxy, I would like no default and if a timeout isn't provided then no annotation is added